main.go

```diff
@@ -276,41 +281,66 @@ func downloadSourcePaths(sourcePaths []string, destPath string, githubRepo GitHu
 
 // Download the specified binary file that was uploaded as a release asset to the specified GitHub release.
 // Returns the path where the release asset was downloaded.
```

Comment:
Can you update this comment so that it captures the fact that we now:

- match the regex against all the assets on the release tag
- use go routines to download each one.